packages/terra-consumer-nav/src/components/user-profile/UserProfile.scss

```diff
@@ -58,10 +58,6 @@
     text-align: left;
     width: inherit;
 
-    @media screen and (max-width: $terra-medium-breakpoint) {
-      margin-top: 0;
-    }
-
```

Comment:
why was this removed?

Reply:
UX decision to keep the separation between nav items and profile link via the 3px margin-top punch through